api/db/changelog/db.changelog-109-dataset-prepackaged-conceptsets-nullConstraint.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="nsaxena" id="db.changelog-109-dataset-prepackaged-conceptsets-nullConstraint">
+    <addNotNullConstraint columnDataType="tinyint"
+            columnName="prePackagedConceptSet"
```

Comment:
Should this column be renamed `pre_packaged_concept_set_id`?

Reply:
should we tackle that in another PR?

Reply:
I'd most likely do both in one PR, though you could make it two separate migrations. Not required, but maybe file a ticket for this and any other columns that don't match the convention.